setup.py

```diff
@@ -2,8 +2,8 @@
 
 extras_require = {
     "repr": ["pandas >=1.2"],
-    "io": ["networkx >=2.8", "scipy >=1.8", "awkward >=1.9"],
-    "viz": ["matplotlib"],
+    "io": ["networkx >=2.8", "scipy >=1.8", "awkward >=1.9", "sparse >=0.12"],
+    "viz": ["matplotlib"],  # Min required version?
```

Comment:
Could we set this to `3.6` for now? And then work backward if it causes problems?

Reply:
Sure, but let's do `matplotlib >=3.5`. Also, this metadata has been moved to `pyproject.toml`, so it'll need to be added there to fix the merge conflict (sorry about that!).

Reply:
OK, I will discard these changes and re-intro them to pyproject.tom after syncing to the main.